src/__init__.py

```
@@ -0,0 +1,96 @@
+		from .bot_instance import get_bot_instance
+		bot = get_bot_instance()
+		if bot is None:
+			return
+		sess = getattr(bot, "session", None)
+		if sess is None:
+			return
+		try:
+			# close() is safe to call; prefer sync close to avoid
+			# requiring an event loop at shutdown
+			sess.close()
+		except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)